CHANGELOG.md

```diff
@@ -5,7 +5,17 @@ The Singularity Project has been
 and re-branded as Apptainer.
 For older changes see the [archived Singularity change log](https://github.com/apptainer/singularity/blob/release-3.8/CHANGELOG.md).
 
-## Changes for v1.3.x
+## v1.3.2 - \[2024-05-28\]
+
+### Security fix
+
+- Included a fix for
+  [security advisory GHSA-6wvf-f2vw-3425](https://github.com/advisories/GHSA-6wvf-f2vw-3425)
```

Comment:
I think it would be good to make it clear in the text that it is in a dependent library so people don't need to follow the link to know that.  Also while you're changing it, it would be helpful to use the CVE number in the link text instead of the GHSA number.